### PR TITLE
Test Unit Testing of EntryButton, SnapShot for EntryButton and Cypress flow for navigating pages

### PR DESCRIPTION
**Pull request by BobVerheij**

### CODE REVIEW

File: db.json [hunk 1]
```diff
@@ -778,4 +778,4 @@
       "profilePicture": ""
     }
   ]
-}
\ No newline at end of file
+}
```
Comment (RubenWerdmuller) [inline db.json:781]: Wasn't this file added to the `.gitignore?`

File: src/components/MenuNavigation.tsx [hunk 2]
```diff
@@ -11,7 +11,7 @@ interface NavProps {
 }
 function MenuNavigation({ isOpen, activePage }: NavProps) {
   return (
-    <Styled.MenuNavigation isOpen={isOpen}>
+    <Styled.MenuNavigation data-cy="MenuNavigation" isOpen={isOpen}>
```
Comment (RubenWerdmuller) [inline src/components/MenuNavigation.tsx:14]:
```suggestion
    <Styled.MenuNavigation data-cy="menu-navigation" isOpen={isOpen}>
```

File: src/components/entrybutton/EntryButton.tsx [hunk 3]
```diff
@@ -3,17 +3,29 @@ import React from "react";
...
 }
-function EntryButton({ label, type = "AddButton", width = 300 }: EntryButtonProps) {
+function EntryButton({
+  handleClick,
+  isVisible = true,
+  label,
+  type = "AddButton",
+  width = 300,
+}: EntryButtonProps) {
   return (
```
Comment (RubenWerdmuller) [inline src/components/entrybutton/EntryButton.tsx:19]: Implement the actual `Button` you use within the app

File: src/components/entrybutton/__test__/EntryButton.test.tsx [hunk 4]
```diff
@@ -0,0 +1,26 @@
+/* eslint-disable import/no-extraneous-dependencies */
+// import { mount } from "enzyme";
+import React from "react";
+import renderer from "react-test-renderer";
+
+import EntryButton from "../EntryButton";
+
+// it("should result in the entryButton being invisible", () => {
+//   EntryButton
+// });
```
Comment (RubenWerdmuller) [inline src/components/entrybutton/__test__/EntryButton.test.tsx:10]: remove comments

File: src/services/time-entries/__test__/getTimeEntries.test.tsx [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+  const unmockedFetch = global.fetch;
+  global.fetch = jest.fn().mockImplementation(() => mockFetchPromise);
+
+  getTimeEntries();
+
+  expect(global.fetch).toHaveBeenCalledTimes(1);
+  expect(
+    global.fetch,
+  ).toHaveBeenCalledWith(
+    "https://my-json-server.typicode.com/humanoidsbv/team-awesome-bob/time-entries?_sort=startTime",
```
Comment (RubenWerdmuller) [inline src/services/time-entries/__test__/getTimeEntries.test.tsx:28]: In our case you would want to implement the localhost variant as well!

**Review by RubenWerdmuller — CHANGES_REQUESTED** [review]
👌

File: src/services/time-entries/__test__/getTimeEntries.test.tsx (hunk 5 above)
Comment (BobVerheij) [reply src/services/time-entries/__test__/getTimeEntries.test.tsx]: Do I still need to implement now?

**Pull request merged**